src/Microsoft.Azure.SignalR.Common/RestClients/SignalRServiceRestClientHelper.cs

```diff
@@ -0,0 +1,36 @@
+
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class SignalRServiceRestClientHelper
```

Comment:
how about a `RestClientBuilder` with `WithConnectionString` and `WithProductInfo`? Is this the one to be used by endusers? no need to expose `handlers`?

Reply:
This one are internal class and are not exposed to end users. End users don't have to create REST client themselves.

Reply:
Let's give the client a larger scope, for example, as the convenience layer to be added into azure sdk. Take an example of other RestClients, for example, this one https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Secrets#create-secretclient